_docs/operations/encryption.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: guide
+title: StorageOS Docs - Encryption
+anchor: operations
+module: operations/encryption
+---
+
+# Encryption
+
+Volumes can be encrypted when they are created using the
+`storageos.com/encryption label`. The labels can be passed to StorageOS using
+PVCs or you can directly create volumes using the StorageOS CLI or GUI with the
+encryption label applied. 
+
+For more in depth discussion of how encryption works please see the [Encryption
+concepts](/docs/concepts/encryption) page.
+
+## Required labels
+
+The `storageos.com/encryption` label must be applied to the volume when it is
+created. The encryption status of a volume cannot be changed after a volume has been
+created.
+
+You can pass the label using a PVC
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc0002
+  labels:
+    "storageos.com/encryption": "true"
+  annotations:
+    volume.beta.kubernetes.io/storage-class: fast
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+You can also pass the encryption label when creating volumes using the CLI
+```bash
+$ storageos volume create encrypted --label storageos.com/encryption=true
+```
+
+You can also add the encryption label when creating a volume with the GUI
+
+## Backing up Secrets
+
+The keys that are used to encrypt a volume are stored in a Kubernetes secret.
```

Comment:
I'd add:

StorageOS generates the keys to encrypt data. See (link to concepts) for more details. Once the keys are generated, are stored in Kubernetes Secrets. These keys are only available inside the cluster and are completely private. Therefore StorageOS support team does not have access to the keys at any moment, as such if the keys are lost the volume **cannot** be decrypted.